tests/CMakeLists.txt

```diff
@@ -162,6 +162,7 @@ foreach(py_test_module
     system_tests_open_properties
     ${SYSTEM_TESTS_HTTP2}
     system_tests_http1_adaptor
+    system_tests_http1_over_tcp
```

Comment:
This test needs to be disabled on macOS, because it depends on raw-connections functionality in Proton. That's not available in the libuv proactor. I filled a Jira for it before I realized the test is simply incompatible with macOS https://issues.apache.org/jira/browse/DISPATCH-2042.

Please add test name to

https://github.com/apache/qpid-dispatch/blob/db30cb043bece740b5273b137057aa67b161ed50/.travis.yml#L139-L140